### ISSUE TITLE
Unable to copy single field

### ISSUE THREAD

**Issue by ummspmacdon** [microsoft/azuredatastudio#13780]
When i try to right-click and copy a single result field (ex. image_id) and then paste it anywhere, but usually into a new query, it doesn't paste anything.  I've tried highlighting the field and using CTRL-C/CTRL-P, but that doesn't work either.

Is it one of my setting?

**Comment by alanrenmsft** [comment]
@ummspmacdon thanks for reporting the issue, I will look into it, meanwhile you can use the context menu (right click on the cell to bring it up) to access the copy menu item. the feature is there, but somehow the CTRL+C shortcut key is not working for a single cell selection.

also, when you say CTRL+P, I suppose you mean CTRL+V for pasting. right?

**Comment by ummspmacdon** [comment]
Allen,

The context menu (Control-C and Control-V) are not working either.  If I highlight a cell in the results window and right-click, no menu comes up.  If I highlight a cell in the results window and press Control-C and then try to paste (Yes, you are correct – Control-V instead of Control-P), the cell value does not get pasted.  What gets pasted is the last item I copied to the clipboard – it could be anything from an email to a spreadsheet value to a URL…whatever was last copied…..not the cell value.   It’s like it never even recognizes that I tried to copy the cell.

Paul

-------------------------------------------------------------------------------------------------------------------------------
Paul T. MacDonald | Archive/Decommissioning Project | email: pmacdonald1@umm.edu<mailto:pmacdonald1@umm.edu>  |  Phone:  410.353.8363

**Comment by ummspmacdon** [comment]
Another issue that I think coincides with this is the fact that I no longer see the icons along the right-hand side of the results window that allows me to export to Excel, or text or json…..

To wrap it up, I can’t copy a single cell of the results, I can’t copy any portion of the results grid and I can’t export to a useful format……

Paul

-------------------------------------------------------------------------------------------------------------------------------
Paul T. MacDonald | Archive/Decommissioning Project | email: pmacdonald1@umm.edu<mailto:pmacdonald1@umm.edu>  |  Phone:  410.353.8363

From: MacDonald, Paul
Sent: Wednesday, December 16, 2020 7:19 AM
To: microsoft/azuredatastudio <reply@reply.github.com>
Subject: RE: [microsoft/azuredatastudio] Unable to copy single field (#13780)

Allen,

The context menu (Control-C and Control-V) are not working either.  If I highlight a cell in the results window and right-click, no menu comes up.  If I highlight a cell in the results window and press Control-C and then try to paste (Yes, you are correct – Control-V instead of Control-P), the cell value does not get pasted.  What gets pasted is the last item I copied to the clipboard – it could be anything from an email to a spreadsheet value to a URL…whatever was last copied…..not the cell value.   It’s like it never even recognizes that I tried to copy the cell.

Paul

-------------------------------------------------------------------------------------------------------------------------------
Paul T. MacDonald | Archive/Decommissioning Project | email: pmacdonald1@umm.edu<mailto:pmacdonald1@umm.edu>  |  Phone:  410.353.8363

**Comment by alanrenmsft** [comment]
@ummspmacdon what is your version of ADS and what OS are you using? I just tried on Windows and macOS, both the latest stable and insider, I couldn't reproduce the issue.

![image](https://user-images.githubusercontent.com/13777222/102553463-bc567280-4077-11eb-9fac-fbff0fffeb3b.png)

do you see any errors in the console? (Help menu -> Toggle Developer tools -> console tab)
![image](https://user-images.githubusercontent.com/13777222/102553593-f162c500-4077-11eb-9632-e465a960b0a4.png)

**Comment by ummspmacdon** [comment]
Version: 1.25.1 (user setup)
Commit: e7eacc32c0d20bd4da843320c3bfac231b308c98
Date: 2020-12-11T01:59:24.271Z
VS Code: 1.48.0
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.17763

Paul

-------------------------------------------------------------------------------------------------------------------------------
Paul T. MacDonald | Archive/Decommissioning Project | email: pmacdonald1@umm.edu<mailto:pmacdonald1@umm.edu>  |  Phone:  410.353.8363

**Comment by alanrenmsft** [comment]
@ummspmacdon how about console logs? Could you please also attach a screenshot of the query results area? please make sure no sensitive data is included.

**Comment by alanrenmsft** [comment]
@ummspmacdon when attaching screenshots, you might have to open github in browser, If I remember correctly, images in the email reply won't show up here.

**Comment by ummspmacdon** [comment]
[Graphical user interface, text, application  Description automatically generated]

[A picture containing text, computer  Description automatically generated]

Paul

-------------------------------------------------------------------------------------------------------------------------------
Paul T. MacDonald | Archive/Decommissioning Project | email: pmacdonald1@umm.edu<mailto:pmacdonald1@umm.edu>  |  Phone:  410.353.8363

**Comment by ummspmacdon** [comment]
Alan,

After I sent those screenshots, I was looking at the Developer Tools to get the Console log.  When I opened it, there were hundreds of entries.  Some said ‘dirty’ profile, some mention rendering etc.  When I tried to capture the log to save it, it went away.  When I closed out the app to restart it hoping to get a clean log, everything returned to normal.  The icons are back.  I can select and copy results as before.  I don’t know how I got into such a state for the last few days or how I fixed it this morning, but it’s working and I’m happy.

I guess we can close the ticket and blame it on user error!  Thanks for the time and effort.

Paul

-------------------------------------------------------------------------------------------------------------------------------
Paul T. MacDonald | Archive/Decommissioning Project | email: .<mailto:pmacdonald1@umm.edu>  |  Phone:  410.353.8363
